jetbrains-gateway/src/software/aws/toolkits/jetbrains/gateway/CawsConnectionProvider.kt

```diff
@@ -103,10 +107,31 @@ class CawsConnectionProvider : GatewayConnectionProvider {
             return null
         }
 
+        val currentConnection = ToolkitConnectionManager.getInstance(null).activeConnectionForFeature(CodeCatalystConnection.getInstance())
+            as AwsBearerTokenConnection?
         val ssoSettings = connectionParams.ssoSettings ?: SsoSettings(SONO_URL, SONO_REGION)
+        if (connectionParams.ssoSettings?.startUrl != "-1" && connectionParams.ssoSettings?.region != "-1") {
```

Comment:
ssoSettings is not null here

Reply:
ssoSettings is null during local dev env creation right?